src/qml/pages/ChatPage.qml

```diff
@@ -97,7 +97,31 @@ Page {
                     dropdown: true
                     currentIndex: -1
 
-                    Action {
+                    ContextMenuItem {
+                        text: qsTr("Call")
+                        iconName: "Make-Call"
+                        onTriggered: console.log("Chat deletion isn't implemented")
+                    }
+
+                    ContextMenuSeparator {
+                    }
+
+                    ContextMenuItem {
+                        text: qsTr("Chat info")
+                        onTriggered: console.log("Chat deletion isn't implemented")
+                    }
+
+                    ContextMenuItem {
+                        text: qsTr("Mute notifications")
+                        onTriggered: console.log("Chat deletion isn't implemented")
```

Comment:
Hide all menu items that have no implementation, develop has working code only

Reply:
I have made them visible: false, but it doesn't work

Reply:
I will check this

Reply:
Item visibility works good in stdChatContextMenu. I don't see the problem @dmitry-zezik